File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountapproval/SavingsAccountApprovalMvpView.java
package com.mifos.mifosxdroid.online.savingsaccountapproval;

import com.mifos.api.GenericResponse;
import com.mifos.mifosxdroid.base.MvpView;

/**
 * Created by Rajan Maurya on 09/06/16.
 */
public interface SavingsAccountApprovalMvpView extends MvpView {

    void showUserInterface();

    void showSavingAccountApprovedSuccessfully(GenericResponse genericResponse);

    void showError(String s);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountapproval/SavingsAccountApprovalMvpView.kt
package com.mifos.mifosxdroid.online.savingsaccountapproval

import com.mifos.api.GenericResponse
import com.mifos.mifosxdroid.base.MvpView

/**
 * Created by Rajan Maurya on 09/06/16.
 */
interface SavingsAccountApprovalMvpView : MvpView {
    fun showUserInterface()
    fun showSavingAccountApprovedSuccessfully(genericResponse: GenericResponse?)
    fun showError(s: String?)
}